R/nodeCentricSteinerTreeProblem-class.R

```diff
@@ -98,7 +103,7 @@ nodeCentricSteinerTreeProblem <- R6Class("nodeCentricSteinerTreeProblem",
         if( ! (is.logical(private$nodeDT$isTerminal) & all(!is.na(private$nodeDT$isTerminal))) ) stop("isTerminal node attributes *must* all be boolean, with no NA's")
       }
       
-      # check for nodeScore: all -1 if absent, validate if present
+      # check for nodeScore: all -1 if absent, validate if present (This is redundant now that I have added nodeScore to the searchgraph itself)
```

Comment:
If you're happy with the fix above, the following line can be removed.

Reply:
Yes, it makes sense for the nodeScore attribute to be handled at the stage of network checking